core/src/main/scala/water/support/GBMSupport.scala

```diff
@@ -39,6 +41,7 @@ trait GBMSupport {
     * @tparam T H2O Frame Type
     * @return Gradient Boosting Model
     */
+  @DeprecatedMethod("Sparkling Water Algorithm API to train and score H2O models", "3.32")
```

Comment:
Maybe, I would add reference to the `H2OGBM` class.

Reply:
👍